cypress/e2e/smoke/create_tx.cy.js

```diff
@@ -0,0 +1,87 @@
+const SAFE = 'gor:0x04f8b1EA3cBB315b87ced0E32deb5a43cC151a91'
+const EOA = '0xE297437d6b53890cbf004e401F3acc67c8b39665'
+
+describe('Queue a transaction on N/1', () => {
```

Comment:
`N/1` means N owners out of 1? Why not just 1/1 then?

Reply:
I meant 1/N